test/e2e-go/cli/goal/expect/README.md

```diff
@@ -0,0 +1,51 @@
+# Goal Testing with Expect
+
+Expect is a framework for testing command line interfaces (CLI).  It is a extension to the TCL shell designed to automate invoking, interacting with, and validating results of CLIs. 
+ 
+We use expect to test the Algorand Goal CLI.
+
+## Setup 
+
+From the go-algorand root directory, setup the environment and build the binaries as described in the top-level project README.md file.  
+
+#### Initialize the project
+```bash
+git clone https://github.com/algorand/go-algorand
+cd go-algorand
+./scripts/configure_dev.sh
+```
+#### Build the binaries
+```bash
+make clean install
+```
+
+#### Set environment variables
+
+The `GOPATH` should be set to your local Go projects directory. For example:
+
+```bash
+export GOPATH=~/GolandProjects/
```

Comment:
We may want to use something generic like `/path/to/go` here, since not everyone is using goland

Reply:
Updated.